apis/k8ssandra/v1alpha1/k8ssandracluster_types.go

```diff
@@ -411,6 +411,27 @@ type DatacenterOptions struct {
 	ManagementApiAuth *cassdcapi.ManagementApiAuthConfig `json:"managementApiAuth,omitempty"`
 }
 
+// NetworkingConfig is a copy of cass-operator's NetworkingConfig struct. It is copied here to
+// change the HostNetwork field type from bool to *bool, which makes merging 2 values of this struct
+// more intuitive.
+type NetworkingConfig struct {
```

Comment:
This was a bit disappointing, but there is no way we can merge a `bool` field and expect trilean semantic, so I had to copy the struct here and change to `*bool`.

Reply:
Note that this has no impact on the CRD definition.